### PR TITLE
updated README.md

### PR DESCRIPTION
**Pull request by SanyuktaSaha**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,6 +60,8 @@ workflows:
     jobs:
       - cypress/run:
           record: true
+          yarn: true
```
Comment (bahmutov) [inline README.md]: well, these two options are unnecessary and introduce options that are uncommon

**Comment by bahmutov** [comment]
I do like renovate both and tsconfig, but not README example update

File: .circleci/config.yml [hunk 2]
```diff
@@ -5,9 +5,12 @@ jobs:
       - image: circleci/node:10
 
     working_directory: ~/repo
-
     steps:
+
       - checkout
+      - run:
+          name: Update npm
+          command: 'sudo npm install -g npm@latest'
```
Comment (bahmutov) [inline .circleci/config.yml]: no, I don't think this is necessary

File: .vscode/settings.json [hunk 3]
```diff
@@ -1,3 +1,4 @@
 {
-  "editor.formatOnSave": true
-}
\ No newline at end of file
+  "editor.formatOnSave": true,
+  "editor.formatOnPaste": true
```
Comment (bahmutov) [inline .vscode/settings.json]: I don't like formatting on paste

**Comment by KyleTryon** [comment]
@jennifer-shehane sorry this or was sent to CircleCI as a submission for Orbtoberfest. Any PR during the month of October is valid for Hacktoberfest and does not require a hacktoberfest label.

However Digital Ocean has asked (not required) maintainers to mark PRs this month as invalid if a PR should not be considered for Hacktoberfest.

https://hacktoberfest.digitalocean.com/faq#rules

The definition of a "spam" PR is loosely defined. 


This can be ignored if desired.

**Pull request closed without merging**